modules/core/client/components/TimeAgo.js

```diff
@@ -0,0 +1,40 @@
+import moment from 'moment';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+const REFRESH_INTERVAL = 10000; // 10s
+
+/**
+ *  An automatically refreshing time ago component.
+ *
+ *  It refreshes every 10 seconds. If this seems a bit inefficient at some point
+ *  e.g. there are loads of components pointlessly refreshing, there are two things
+ *  we could do:
+ *    1. share the setInterval across all instances
+ *    2. switch to slower updates for further ago dates
```

Comment:
It could also be that `socket.io` type solution works better in some cases instead of polling. But I like how easy polling is to reason about.

Reply:
This is about updating "time from now" display things like from "a few seconds ago" so that a minute later it'll update to say "about a minute ago" (or whatever the moment time ago formatter says).

Reply:
Ahhh, gotcha! 👍